### DISCUSSION TITLE
FN popup

### DISCUSSION THREAD

**Discussion by stefanofabbriiv** [ShankarBUS/ModernFlyouts#335] [Ideas]
Hi,
I'm on Surface Pro, my keyboard allows to turn on/off FN-button too, similar to Caps Lock button. Is there a way to show FN activation/deactivation popup too? As caps lock popup. Thanks

**Comment by ShankarBUS** [comment]
I would give this a big **"N O P E"**. Please see #57 for further details.

**Comment by prakharb5** [comment]
Locking.

@stefanofabbriiv, please now open issues for the same instead of discussions :)
